### PR TITLE
Show partners

### PR DESCRIPTION
**Pull request by mshd**
I want to show unmarried partners:
TODO:
-refresh on change
-fix settings labels
-rename shownRightIds to smt better
-add other option

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ export default class App extends Component {
       hideToggleButton: false,
       showExternalImages: false,
       showFace: false,
+      shownRightIds: "spouse",
```
Comment (ogroppo) [inline src/App.js:41]: what about `rightEntityType`?

File: src/lib/addEntityConnectors.js [hunk 2]
```diff
@@ -26,16 +27,27 @@ export default function addEntityConnectors(entity, propId, options = {}) {
...
     delete entity.rightIds;
   }
   if (options.addLeftIds) entity.leftIds = getClaimIds(entity, SIBLINGS_ID);
   else delete entity.leftIds;
 }
 
-function addRightIds(entity) {
-  const claim = entity.claims[SPOUSE_ID] || []; //cannot use simpleclaims here as only preferred will show up
+function addRightIds(entity, options) {
+  let claim;
```
Comment (ogroppo) [inline src/lib/addEntityConnectors.js:39]:
```suggestion
  let claim = [] //and remove the last `else`?
```

File: src/modals/SettingsModal/SettingsModal.js [hunk 3]
```diff
@@ -158,9 +158,42 @@ export default function SettingsModal({ show, hideModal }) {
           </Dropdown.Menu>
           <Form.Text className="text-muted mt-0">
             If the property or language is available, a second label will be
-            shown, useful when people is known with different names
+            shown, useful when people are known by different names
```
Comment (ogroppo) [inline src/modals/SettingsModal/SettingsModal.js:161]: 🤦 people are awesome

File: src/modals/SettingsModal/SettingsModal.js [hunk 4]
```diff
@@ -158,9 +158,42 @@ export default function SettingsModal({ show, hideModal }) {
...
           </Form.Text>
         </Dropdown>
+        Right to nodes will be
```
Comment (ogroppo) [inline src/modals/SettingsModal/SettingsModal.js:164]: If you forgive me for the sentence above could you write this in english please 😂

File: src/modals/SettingsModal/SettingsModal.js [hunk 5]
```diff
@@ -172,6 +205,7 @@ export default function SettingsModal({ show, hideModal }) {
           />
           <Form.Text className="text-muted pl-4">
             If browsing family trees, the nodes will have a background color
+            (blue for men, red for women)
```
Comment (ogroppo) [inline src/modals/SettingsModal/SettingsModal.js:208]: red or pink?

**Review by ogroppo — CHANGES_REQUESTED** [review]
Great idea!

File: src/lib/addEntityConnectors.js (hunk 2 above)
Comment (mshd) [reply src/lib/addEntityConnectors.js:39]: Yeah.

File: src/modals/SettingsModal/SettingsModal.js (hunk 4 above)
Comment (mshd) [reply src/modals/SettingsModal/SettingsModal.js:164]: Sorry I wanted to code, but my mind was not in English

File: src/App.js (hunk 1 above)
Comment (mshd) [reply src/App.js:41]: Sounds good

File: src/modals/SettingsModal/SettingsModal.js (hunk 5 above)
Comment (mshd) [reply src/modals/SettingsModal/SettingsModal.js:208]: reddish

**Comment by mshd** [comment]
Still can't figure out whats the best way to save the settings, now the LABEL is saved in the settings, ideally the key

**Pull request merged**